composer.json

```diff
@@ -5,7 +5,7 @@
     "license": "MIT",
     "type": "project",
     "require": {
-        "php": "^8.1",
+        "php": "^8.3",
```

Comment:
Please add this upgrade to the deploy notes to remind me to bump the servers before I deploy.